platform/linux-dpdk/test/wrapper-script.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-export ODP_PLATFORM_PARAMS=${ODP_PLATFORM_PARAMS:--n 4 --vdev "crypto_openssl"}
+export ODP_PLATFORM_PARAMS=${ODP_PLATFORM_PARAMS:--n 4 "--vdev \
+"crypto_openssl" --vdev "crypto_null""}
```

Comment:
Will it work w/o crypto_null driver?

Reply:
yes it works.

Reply:
Can you please paste the test log w/o crypto_null somewhere (pastebin or e-mail)? Thanks!

Reply:
all the tests in validations will be skipped because the validation tests works with combination of null algo which is not supported by openssl PMD. If you have a test case that works with valid algos (i.e without null_algo), it should work without an issue.

Reply:
As in fact you are not using crypto_null (except for the lame case of CIPHER_NULL/AUTH_NULL) I suggest to return static capabilities for NULL algos.

Reply:
why static?.  we have the support from dpdk and you just have to initialize it like other PMD's and it is consistent with other PMD's when reading capabilities etc..

Reply:
Because you don't use crypto_null. In fact all code will work perfectly w/o crypto_null, if it did not miss NULL capabilities

Reply:
but I suppose it doesn't hurt the current way of initializing null algos. To change this to static needs some rework to be done and I think it can be updated later by anyone.